pkg/apis/kudo/v1beta1/instance_types.go

```diff
@@ -37,6 +37,23 @@ type InstanceSpec struct {
 	OperatorVersion corev1.ObjectReference `json:"operatorVersion,omitempty"`
 
 	Parameters map[string]string `json:"parameters,omitempty"`
+
+	PlanExecution PlanExecution `json:"planExecution,omitempty"`
```

Comment:
I think I like it in the spec.

Now when I say webhook I know you'll lose all your good mood :smile: but I wonder, how will we do it with the updates (I think we would have the same problem with the annotation).

I assume when PlanExecution is set, we won't allow updates to that (other than ones with force in the future?)

But how will controller clear this? PlanExecution is in progress, so no updates to the spec are allowed, the webhook would somehow have to know that the thing that wants to reset the planExecution is controller? 🤔

Reply:
> the webhook would somehow have to know that the thing that wants to reset the planExecution is controller?

Either that, or clearing `PlanExecution` (which equals to canceling a scheduled PE) is always allowed.